ansible/edusharing-crawler-playbook.yml

```diff
@@ -0,0 +1,80 @@
+---
+- name: Deploy to Edusharing Clusters
+  hosts:
+    - edusharing
+  connection: local
+  tasks:
+  - name: Add Helm Chart Repo dbildungscloud
+    kubernetes.core.helm_repository:
+      name: dbildungscloud
+      repo_url: "https://hpi-schul-cloud.github.io/helm-charts-registry"
+
+  - name: Add Helm Chart Repo ntppool
+    kubernetes.core.helm_repository:
+      name: ntppool
+      repo_url: "https://charts.ntppool.org/"
+
+  - name: Create edusharing-instance secret
```

Comment:
The 3 OnepasswordItems can be rolled out with a loop

Reply:
done :)